src/editors/containers/VideoEditor/components/VideoSettingsModal/components/ErrorSummary.jsx

```diff
@@ -1,40 +1,27 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 
 import { Alert } from '@edx/paragon';
 import { Info } from '@edx/paragon/icons';
 import messages from './messages';
+import { ErrorContext } from '../../../hooks';
 
-export const ErrorSummary = ({
-  error,
-}) => (
-  <Alert
-    icon={Info}
-    show={!Object.values(error).every(val => Object.keys(val).length === 0)}
-    variant="danger"
-  >
-    <Alert.Heading>
-      <FormattedMessage {...messages.validateErrorTitle} />
-    </Alert.Heading>
-    <p>
-      <FormattedMessage {...messages.validateErrorBody} />
-    </p>
-  </Alert>
-);
-
-ErrorSummary.defaultProps = {
-  error: {
-    duration: {},
-    handout: {},
-    license: {},
-    thumbnail: {},
-    transcripts: {},
-    videoSource: {},
-  },
-};
-ErrorSummary.propTypes = {
-  error: PropTypes.node,
+export const ErrorSummary = () => {
+  const error = React.useContext(ErrorContext);
+  return (
+    <Alert
+      icon={Info}
+      show={!Object.values(error).every(val => Object.keys(val[0]).length === 0)}
```

Comment:
I think it might be better to separate the function inside of `every()` as its own and call it here. It is a little difficult to read what is happening here.  It would also allow for a bit more variable naming to make the readability clearer.

Reply:
I would also change the length check from `=== 0` to `!==0` and you won't have to put `!` at the beginning of the line.

Reply:
looks good!